### PR TITLE
Add rejection of custom logits processors.

### PR DESCRIPTION
**Pull request by viktorpusTT**
## Purpose

Add rejection of custom logits processors which was accidentally removed in the recent upstream pull. 

SamplingParams no longer has a logits_processors field of its own. The custom processors are now configured at engine/model initialization time via logits_processors=... on LLM / ModelConfig, not attached directly to each request’s SamplingParams.

## Test Result

https://github.com/tenstorrent/tt-metal/actions/runs/24822901197

---
<details>
<summary> Essential Elements of an Effective PR Description Checklist </summary>

- [ ] The purpose of the PR, such as "Fix some issue (link existing issues this PR will resolve)".
- [ ] The test plan, such as providing test command.
- [ ] The test results, such as pasting the results comparison before and after, or e2e results
- [ ] (Optional) The necessary documentation update, such as updating `supported_models.md` and `examples` for a new model.
</details>

### CODE REVIEW

**Comment by tchedaTT** [comment]
Could we actually support these? We already have some infra in place by gathering the builtin logitsprocs (mintokens,logitsbias,minp) through input_batch.sampling.logitsprocs and host_only_sample_params["logitsprocs"]. What are we missing/what's the blocker?

**Review by tchedaTT — APPROVED** [review]

**Comment by viktorpusTT** [comment]
I've tested that custom logits processors actually work, no need to reject.
